src/test/java/net/openhft/chronicle/queue/impl/single/stress/RollCycleMultiThreadStressTest.java

```diff
@@ -165,6 +165,7 @@ public void stress() throws Exception {
         if (PRETOUCH) {
             pretoucherThread = new PretoucherThread(file);
             executorServicePretouch.submit(pretoucherThread);
+            expectException("touchPage failed");
```

Comment:
Do we still need this?

Reply:
Yes. RollCycleMultiThreadStressPretouchEATest is now in QE and descends from this